content/300-guides/200-deployment/201-serverless/500-deploy-to-netlify.mdx

```diff
@@ -237,44 +67,45 @@ You can now create your Netlify site and configure continuous deployments for it
 
    1. Select `Create & configure a new site`.
    2. If prompted to select your team, pick the one in which Netlify will create the site.
-   3. Give the site a unique name.
-   4. Authorize Netlify with your GitHub account.<br />
-      Netlify uses your GitHub account to configure notification webhooks and deploy keys. It uses these to configure the continuous integration for the app.
-   5. _Your build command_: `npm run build`
-   6. _Directory to deploy_: `public`
+   3. Give the site a unique name, or use the default. It can be updated in the future.
+   4. Authorize Netlify with your GitHub account. Netlify uses your GitHub account to configure notification webhooks and deploy keys. It uses these to configure the continuous integration for the app.
+   5. _Your build command_: The command you use to build your project. Common examples include `npm run build`, `yarn run build`, `next build`, etc.
+   6. _Directory to deploy_: Generally, the directory that your `build` command outputs to. Common examples include `dist/`, `build/`, `.next`, etc.
+   7. Allow Netlify to create a `netlify.toml` file.
```

Comment:
Is this the up to date flow?

Reply:
Yeah, I ran through this flow. I'll double check.